shaunkendrone/call-center-ticketing-system
# Ft update ticket

What does this PR do?
Allows agents to modify the details of an existing ticket in the call center ticket system.

Description of task to be completed?
It provides the capability to make changes to attributes of a ticket such as the status, priority, issue description, and category. This feature enables agents to keep the ticket information up to date.

`How should this be manually tested?
- Ensure there are existing tickets in the system that can be updated.
- The system should prompt for the ticket ID of the ticket to be updated. Enter the ID of an existing ticket that needs to be modified.
- Verify that the system correctly identifies the ticket with the provided ID and retrieves its current details for editing.
- Proceed to modify the desired attributes of the ticket
- After modifying the agent name, the ticket should be updated successfully.
- Ensure that the system successfully applies the updates to the ticket and reflects the new values for the modified attributes. This can be done by choosing "Get All Tickets" from the menu and checking for the updated ticket.
- Test other cases of ticket id by entering a non existing ticket id.

Screenshots
![Screenshot 2023-07-08 001712](https://github.com/shaunkendrone/call-center-ticketing-system/assets/98257312/effe0f0c-1b4d-4c73-96a5-406863be1966)
![Screenshot 2023-07-08 003116](https://github.com/shaunkendrone/call-center-ticketing-system/assets/98257312/c95b9344-7ee1-4698-8dac-b088cc214d0a)
![Screenshot 2023-07-08 003132](https://github.com/shaunkendrone/call-center-ticketing-system/assets/98257312/5f6aadcd-4c37-446b-920b-5ce9427e1e5d)
